jsk_arc2017_baxter/euslisp/lib/util.l

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env roseus
+
+(require "package://jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/util.l")
+
+(ros::load-ros-manifest "jsk_gui_msgs")
+(ros::load-ros-manifest "jsk_recognition_msgs")
+
+(defun opposite-arm (arm) (if (eq arm :larm) :rarm :larm))
```

Comment:
In my oppinion, 
```
(require "package://jsk_arc2017_baxter/euslisp/lib/util.l")  ;; with no namespace
```
and
```
(require "package://jsk_arc2017_baxter/euslisp/lib/util.l")  ;; with namespace
(in-package "jsk_arc2017_baxter) 
```
make a similar result, so we don't have to use namespace if we use `in-package`.